openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
```

Comment:
This is a pattern often found in the codebase to classify the methods of a class (see [here](https://github.com/openfisca/openfisca-core/blob/f9fa2813b3790115ee8663a31eb939931eb264b9/openfisca_core/entities.py#L26) or [there](https://github.com/openfisca/openfisca-core/blob/f9fa2813b3790115ee8663a31eb939931eb264b9/openfisca_core/entities.py#L87) and [there](https://github.com/openfisca/openfisca-core/blob/f9fa2813b3790115ee8663a31eb939931eb264b9/openfisca_core/entities.py#L73).

Sure, in this case it's abusive! I'll remove it.